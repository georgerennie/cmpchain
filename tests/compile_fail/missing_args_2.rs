use cmpchain::chain;
fn main() { chain!(1 < ); }