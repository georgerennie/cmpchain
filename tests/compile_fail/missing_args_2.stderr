error: Expected two arguments for "<"
 --> $DIR/missing_args_2.rs:2:13
  |
2 | fn main() { chain!(1 < ); }
  |             ^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
